tkg/client/cluster.go

```diff
@@ -738,13 +738,46 @@ func (c *TkgClient) ConfigureAndValidateWorkloadClusterConfiguration(options *Cr
 		return err
 	}
 
+	if err = c.ConfigAndValidateExtraArgs(); err != nil {
+		return err
+	}
+
 	if err = c.ValidateKubeVipLBConfiguration(TkgLabelClusterRoleWorkload); err != nil {
 		return NewValidationError(ValidationErrorCode, err.Error())
 	}
 
 	return c.ValidateSupportOfK8sVersionForManagmentCluster(clusterClient, options.KubernetesVersion, skipValidation)
 }
 
+func (c *TkgClient) ConfigAndValidateExtraArgs() error {
```

Comment:
seems we only do validation for ExtraArgs in this func, what about renaming the func to avoid confusion?

Reply:
We also set empty string if not found